This test is modified so we use a meaningful base fee calculation that shows
fee adjustment. If we use block number 1, the parent will be genesis and the
current base fee will be minBaseFee.
The ability to set minBaseFee was added to the test, since the default minBaseFee
is too high for the test tx to work.

--- a/cmd/evm/testdata/25/env.json	2023-08-25 07:34:20
+++ b/cmd/evm/testdata/25/env.json	2023-08-24 14:17:32
@@ -6,6 +6,7 @@
   "parentBaseFee": "0x500",
   "parentGasUsed": "0x0",
   "parentGasLimit": "0x750a163df65e8a",
-  "currentNumber": "1",
-  "currentTimestamp": "1000"
+  "currentNumber": "2",
+  "currentTimestamp": "1",
+  "minBaseFee": "0x1"
 }
